### ISSUE TITLE
🐞 Installation instructions out of date?

### ISSUE THREAD

**Issue by BevanR** [dominique-mueller/no-skipped-tests#48]
After following the installation instructions from the README, `npm run pretest` fails;

```
$ npm list no-skipped-tests
my-project@1.0.125 ~/projects/my-project
└── no-skipped-tests@1.0.0

$ npm run pretest

> my-project@1.0.125 pretest ~/projects/my-project
> no-skipped-tests

sh: no-skipped-tests: command not found
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! my-project@1.0.125 pretest: `no-skipped-tests`
npm ERR! spawn ENOENT
npm ERR!
npm ERR! Failed at the my-project@1.0.125 pretest script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     ~/.npm/_logs/2019-05-06T02_17_06_493Z-debug.log
```

**Comment by itsdevdom** [comment]
Yes, you're right, the README has a copy-paste issue. You can of course run `npm run pretest` but only if that script is set up. Running the tool itself without that script can be done using `npm run no-skipped-tests`.

I'll fix it.

**Comment by itsdevdom** [comment]
Fixed and published with version 2.0.0!
